### PR TITLE
Update libre-fix-bt-connection-issues.md

### PR DESCRIPTION
**Pull request by PieterDeBruijn**
-more consistant layout of this readme file
-more consistant naming (xdrip = xDrip)
-fixed typos

### CODE REVIEW

File: Documentation/libre-fix-bt-connection-issues.md [hunk 1]
```diff
@@ -1,15 +1,11 @@
-
-
 # If you have a lot of missed readings:
-
 You can try other settings:
-
```
Comment (tolot27) [inline Documentation/libre-fix-bt-connection-issues.md:6]: Empty lines after headings are not required according to the spec, but improving the readability in plain text mode. Please add/keep it.

**Review by tolot27 — DISMISSED** [review]
Thanks for you contribution.
Are you willing to support us writing a documentation or improving the wiki? @Navid has a lot of experience with Dexcom but not with Libre and we are looking for more contributions to the Libre docs.

**Review by tolot27 — APPROVED** [review]

**Comment by Navid200** [comment]
I don't think that file will be seen by many people who have difficulty and are inexperienced.

But, as has been mentioned already, would you be willing to help us with documentation?

**Comment by PieterDeBruijn** [comment]
Yes, I already sent you an email this morning. I would be happy to help in that way! Kind regards, Pieter

**Comment by Navid200** [comment]
We are different individuals.  Hopefully, one of us has got your email.  I haven't.

**Comment by PieterDeBruijn** [comment]
> We are different individuals.  Hopefully, one of us has got your email.  I haven't.

Apparently more Navid's here on GitHub, sorry, so here's my email;
Hi Navid,

I want to contribute to xDrip, we are using Libre 2 (NFC and Bluetooth) sensor and Dexcom G6. I got a message asking me to help improve the Libre documentation. Could you point me in the right direction?
Maybe have a chat and have a virtual beer.

Thanks!

Pieter
https://github.com/PieterDeBruijn
mail@pieterdebruijn.nl
Signal: +31627028244

**Comment by tolot27** [comment]
@PieterDeBruijn I've removed your message because it contains your phone number. Navid got it automatically via e-mail.

**Comment by tolot27** [comment]
> Yes, I already sent you an email this morning. I would be happy to help in that way! Kind regards, Pieter

That is fantastic! Thank you for your support.

**Comment by Navid200** [comment]
I have invited you to join a room in Gitter so that we can talk.  Please let me know if you have not received an invitation.

**Pull request merged**